### DISCUSSION TITLE
EssentialsX-Discord: Broadcast a message to discord with a command.

### DISCUSSION THREAD

**Discussion by JaayTheLoft** [EssentialsX/Essentials#4472] [Ideas and Feature Suggestions] [answered]
### What am I suggesting?

I am suggesting a new command for the **Discord EssentialsX plugin**, this command would allow a person with proper permissions to broadcast a message to a channel in their discord through the bot.

The imagined command would be something like:
**/ediscord broadcast (#channel) (message)**

So if I did _/ediscord broadcast #general The event will begin soon_,
the bot would message on discord "The event will begin soon"

### How will this feature be useful?
This feature would be good to be used by server administrators to announce events or announcements instantly, or even to be used with other plugins to send messages instantly to the channels. For example, for me personally, I would be using a custom command from a custom plugin & use EssentialsX's discord integration to broadcast a message to the discord whenever a user needs assistance from staff. When someone runs /req (a custom command from the custom plugin), it will run "/ediscord broadcast #staff [User] needs assistance at XYZ" through the console, and the message will be sent to discord through EssentialsX's discord integration.

This could also be good to send in-game reports to a certain channel for staff to review on the discord.

### Conclusion
I can overall see this being a very useful feature.

This plugin is already extremely helpful and lightweight, and I prefer it a lot over other Discord integration plugins out there, and many other of the Discord-Minecraft integration plugins have the feature that I am suggesting today, and overall, I hope it can be implemented with EssentialsX. You guys are an awesome and trustworthy developer team, and I prefer your plugins over all others.

**Comment by JRoy** [comment, marked as answer]
This will be added in #4541
